### PR TITLE
Switch perf to studies

### PR DESCRIPTION
**Pull request by atuchin-m**

### CODE REVIEW

**Review by goodov — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -23,9 +24,6 @@ jobs:
       - name: Install requirements
         run: pip install -r seed/requirements.txt
 
-      - name: Serialize seed
-        run: python seed/serialize.py seed/seed.json
-
       - name: Generate testing_fieldtrials.json (HEAD)
```
Comment (goodov) [inline .github/workflows/test.yml:27]: let's rename this workflow file to be something like `test-testing-fieldtrials-gen.yml`, and also rename `name: Test Python Generator` to a better suitable name.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     paths:
       - 'seed/**'
+      - 'studies/**'
```
Comment (goodov) [inline .github/workflows/test.yml:7]: let's also add `- 'src/**'`

**Review by goodov — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (atuchin-m) [reply .github/workflows/test.yml:28]: done.

File: .github/workflows/test.yml (hunk 2 above)
Comment (atuchin-m) [reply .github/workflows/test.yml:7]: added.

**Review by goodov — APPROVED** [review]

**Pull request merged**